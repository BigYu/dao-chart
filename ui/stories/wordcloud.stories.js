import React from 'react';
import { storiesOf } from '@storybook/react';
import WordCloud from '../src/component/wordcloud';

const data = [
  { value: 115835, name: 'life' },
  { value: 78562, name: 'photography' },
  { value: 62453, name: 'steemit' },
  { value: 45831, name: 'blog' },
  { value: 38690, name: 'art' },
  { value: 32495, name: 'bitcoin' },
  { value: 29032, name: 'nature' },
  { value: 27722, name: 'cryptocurrency' },
  { value: 27675, name: 'story' },
  { value: 25752, name: 'travel' },
  { value: 24354, name: 'busy' },
  { value: 23617, name: 'news' },
  { value: 22315, name: 'steem' },
  { value: 21568, name: 'funny' },
  { value: 20074, name: 'writing' },
  { value: 19844, name: 'introduceyourself' },
  { value: 18490, name: 'indonesia' },
  { value: 17766, name: 'food' },
  { value: 16985, name: 'photo' },
  { value: 16761, name: 'spanish' },
  { value: 16632, name: 'crypto' },
  { value: 16433, name: 'aceh' },
  { value: 16345, name: 'money' },
  { value: 14900, name: 'love' },
  { value: 14315, name: 'dtube' },
  { value: 13415, name: 'kr' },
  { value: 12226, name: 'new' },
  { value: 11889, name: 'meme' },
  { value: 11603, name: 'video' },
  { value: 11310, name: 'music' },
  { value: 10737, name: 'blockchain' },
  { value: 9595, name: 'cervantes' },
  { value: 9575, name: 'health' },
  { value: 8545, name: 'fun' },
  { value: 8464, name: 'entertainment' },
  { value: 7916, name: 'poetry' },
  { value: 7629, name: 'steepshot' },
  { value: 7616, name: 'cn' },
  { value: 7558, name: 'nigeria' },
  { value: 7499, name: 'technology' },
  { value: 7459, name: 'dmania' },
  { value: 7434, name: 'science' },
  { value: 7191, name: 'philippines' },
  { value: 6367, name: 'contest' },
  { value: 6231, name: 'politics' },
  { value: 6112, name: 'colorchallenge' },
  { value: 5960, name: 'kr-newbie' },
  { value: 5716, name: 'drawing' },
  { value: 5567, name: 'history' },
  { value: 4845, name: 'motivation' },
  { value: 4722, name: 'utopian-io' },
  { value: 4627, name: 'animals' },
  { value: 4606, name: 'venezuela' },
  { value: 4542, name: 'esteem' },
  { value: 4462, name: 'gaming' },
  { value: 4392, name: 'ethereum' },
  { value: 4267, name: 'education' },
  { value: 4147, name: 'beauty' },
  { value: 4146, name: 'tr' },
  { value: 4051, name: 'animal' },
  { value: 3909, name: 'stach' },
  { value: 3884, name: 'deutsch' },
  { value: 3859, name: 'philosophy' },
  { value: 3837, name: 'sport' },
  { value: 3824, name: 'whalepower' },
  { value: 3699, name: 'creativity' },
  { value: 3672, name: 'community' },
  { value: 3647, name: 'dlive' },
  { value: 3614, name: 'sports' },
  { value: 3500, name: 'promo-steem' },
  { value: 3482, name: 'trading' },
  { value: 3414, name: 'world' },
  { value: 3368, name: 'untalented' },
  { value: 3346, name: 'introducemyself' },
  { value: 3345, name: 'ico' },
  { value: 3307, name: 'youtube' },
  { value: 3198, name: 'sevendaybnwchallenge' },
  { value: 3176, name: 'family' },
  { value: 3163, name: 'natural' },
  { value: 3097, name: 'flower' },
  { value: 3091, name: 'ksi' },
  { value: 2985, name: 'india' },
  { value: 2964, name: 'photofeed' },
  { value: 2959, name: 'btc' },
  { value: 2925, name: 'poem' },
  { value: 2911, name: 'adventure' },
  { value: 2892, name: 'steemph' },
  { value: 2843, name: 'smartphonephotography' },
  { value: 2843, name: 'vincentb' },
  { value: 2783, name: 'inspiration' },
  { value: 2776, name: 'painting' },
  { value: 2760, name: 'castellano' },
  { value: 2731, name: 'fiction' },
  { value: 2643, name: 'business' },
  { value: 2634, name: 'religion' },
  { value: 2548, name: 'freedom' },
  { value: 2492, name: 'humor' },
];


storiesOf('Wordcloud Chart', module)
  .add('wordcloud chart', () => (
    <WordCloud data={data} />
  ));
