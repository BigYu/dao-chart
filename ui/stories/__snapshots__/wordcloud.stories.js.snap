// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot of wordcloud chart 1`] = `
<EchartsReact
  className=""
  echarts={Object {}}
  lazyUpdate={false}
  loadingOption={null}
  notMerge={false}
  onChartReady={[Function]}
  onEvents={Object {}}
  option={
    Object {
      "legend": Object {},
      "series": Array [
        Object {
          "bottom": null,
          "data": Array [
            Object {
              "name": "life",
              "value": 115835,
            },
            Object {
              "name": "photography",
              "value": 78562,
            },
            Object {
              "name": "steemit",
              "value": 62453,
            },
            Object {
              "name": "blog",
              "value": 45831,
            },
            Object {
              "name": "art",
              "value": 38690,
            },
            Object {
              "name": "bitcoin",
              "value": 32495,
            },
            Object {
              "name": "nature",
              "value": 29032,
            },
            Object {
              "name": "cryptocurrency",
              "value": 27722,
            },
            Object {
              "name": "story",
              "value": 27675,
            },
            Object {
              "name": "travel",
              "value": 25752,
            },
            Object {
              "name": "busy",
              "value": 24354,
            },
            Object {
              "name": "news",
              "value": 23617,
            },
            Object {
              "name": "steem",
              "value": 22315,
            },
            Object {
              "name": "funny",
              "value": 21568,
            },
            Object {
              "name": "writing",
              "value": 20074,
            },
            Object {
              "name": "introduceyourself",
              "value": 19844,
            },
            Object {
              "name": "indonesia",
              "value": 18490,
            },
            Object {
              "name": "food",
              "value": 17766,
            },
            Object {
              "name": "photo",
              "value": 16985,
            },
            Object {
              "name": "spanish",
              "value": 16761,
            },
            Object {
              "name": "crypto",
              "value": 16632,
            },
            Object {
              "name": "aceh",
              "value": 16433,
            },
            Object {
              "name": "money",
              "value": 16345,
            },
            Object {
              "name": "love",
              "value": 14900,
            },
            Object {
              "name": "dtube",
              "value": 14315,
            },
            Object {
              "name": "kr",
              "value": 13415,
            },
            Object {
              "name": "new",
              "value": 12226,
            },
            Object {
              "name": "meme",
              "value": 11889,
            },
            Object {
              "name": "video",
              "value": 11603,
            },
            Object {
              "name": "music",
              "value": 11310,
            },
            Object {
              "name": "blockchain",
              "value": 10737,
            },
            Object {
              "name": "cervantes",
              "value": 9595,
            },
            Object {
              "name": "health",
              "value": 9575,
            },
            Object {
              "name": "fun",
              "value": 8545,
            },
            Object {
              "name": "entertainment",
              "value": 8464,
            },
            Object {
              "name": "poetry",
              "value": 7916,
            },
            Object {
              "name": "steepshot",
              "value": 7629,
            },
            Object {
              "name": "cn",
              "value": 7616,
            },
            Object {
              "name": "nigeria",
              "value": 7558,
            },
            Object {
              "name": "technology",
              "value": 7499,
            },
            Object {
              "name": "dmania",
              "value": 7459,
            },
            Object {
              "name": "science",
              "value": 7434,
            },
            Object {
              "name": "philippines",
              "value": 7191,
            },
            Object {
              "name": "contest",
              "value": 6367,
            },
            Object {
              "name": "politics",
              "value": 6231,
            },
            Object {
              "name": "colorchallenge",
              "value": 6112,
            },
            Object {
              "name": "kr-newbie",
              "value": 5960,
            },
            Object {
              "name": "drawing",
              "value": 5716,
            },
            Object {
              "name": "history",
              "value": 5567,
            },
            Object {
              "name": "motivation",
              "value": 4845,
            },
            Object {
              "name": "utopian-io",
              "value": 4722,
            },
            Object {
              "name": "animals",
              "value": 4627,
            },
            Object {
              "name": "venezuela",
              "value": 4606,
            },
            Object {
              "name": "esteem",
              "value": 4542,
            },
            Object {
              "name": "gaming",
              "value": 4462,
            },
            Object {
              "name": "ethereum",
              "value": 4392,
            },
            Object {
              "name": "education",
              "value": 4267,
            },
            Object {
              "name": "beauty",
              "value": 4147,
            },
            Object {
              "name": "tr",
              "value": 4146,
            },
            Object {
              "name": "animal",
              "value": 4051,
            },
            Object {
              "name": "stach",
              "value": 3909,
            },
            Object {
              "name": "deutsch",
              "value": 3884,
            },
            Object {
              "name": "philosophy",
              "value": 3859,
            },
            Object {
              "name": "sport",
              "value": 3837,
            },
            Object {
              "name": "whalepower",
              "value": 3824,
            },
            Object {
              "name": "creativity",
              "value": 3699,
            },
            Object {
              "name": "community",
              "value": 3672,
            },
            Object {
              "name": "dlive",
              "value": 3647,
            },
            Object {
              "name": "sports",
              "value": 3614,
            },
            Object {
              "name": "promo-steem",
              "value": 3500,
            },
            Object {
              "name": "trading",
              "value": 3482,
            },
            Object {
              "name": "world",
              "value": 3414,
            },
            Object {
              "name": "untalented",
              "value": 3368,
            },
            Object {
              "name": "introducemyself",
              "value": 3346,
            },
            Object {
              "name": "ico",
              "value": 3345,
            },
            Object {
              "name": "youtube",
              "value": 3307,
            },
            Object {
              "name": "sevendaybnwchallenge",
              "value": 3198,
            },
            Object {
              "name": "family",
              "value": 3176,
            },
            Object {
              "name": "natural",
              "value": 3163,
            },
            Object {
              "name": "flower",
              "value": 3097,
            },
            Object {
              "name": "ksi",
              "value": 3091,
            },
            Object {
              "name": "india",
              "value": 2985,
            },
            Object {
              "name": "photofeed",
              "value": 2964,
            },
            Object {
              "name": "btc",
              "value": 2959,
            },
            Object {
              "name": "poem",
              "value": 2925,
            },
            Object {
              "name": "adventure",
              "value": 2911,
            },
            Object {
              "name": "steemph",
              "value": 2892,
            },
            Object {
              "name": "smartphonephotography",
              "value": 2843,
            },
            Object {
              "name": "vincentb",
              "value": 2843,
            },
            Object {
              "name": "inspiration",
              "value": 2783,
            },
            Object {
              "name": "painting",
              "value": 2776,
            },
            Object {
              "name": "castellano",
              "value": 2760,
            },
            Object {
              "name": "fiction",
              "value": 2731,
            },
            Object {
              "name": "business",
              "value": 2643,
            },
            Object {
              "name": "religion",
              "value": 2634,
            },
            Object {
              "name": "freedom",
              "value": 2548,
            },
            Object {
              "name": "humor",
              "value": 2492,
            },
          ],
          "height": "100%",
          "left": "center",
          "right": null,
          "rotationRange": Array [
            0,
            0,
          ],
          "rotationStep": 45,
          "shape": "circle",
          "sizeRange": Array [
            8,
            60,
          ],
          "textStyle": Object {
            "emphasis": Object {
              "fontWeight": "bold",
              "opacity": 0.5,
            },
            "normal": Object {
              "color": [Function],
              "fontFamily": "sans-serif",
            },
          },
          "top": "top",
          "type": "wordCloud",
          "width": "100%",
        },
      ],
      "tooltip": Object {
        "show": false,
      },
    }
  }
  opts={Object {}}
  showLoading={false}
  style={Object {}}
  theme={null}
/>
`;
